<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>User Settings</string>
    </property>
    <widget class="QPushButton" name="pushButtonCopy">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QListView" name="listViewDestinationAccounts">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDestinationProfiles">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDestinationInstalls">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOriginProfiles">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listViewOriginAccounts">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOriginInstalls">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectAll">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeselectAll">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Deselect All</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Installations</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Profiles</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>User characters</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuRestore">
     <property name="title">
      <string>Restore</string>
     </property>
     <addaction name="actionRestore_Tranquility"/>
     <addaction name="actionRestore_Singularity"/>
     <addaction name="separator"/>
     <addaction name="actionCustom"/>
    </widget>
    <widget class="QMenu" name="menuBackup">
     <property name="title">
      <string>Backup</string>
     </property>
     <addaction name="actionBackup_Tranquility"/>
     <addaction name="actionBackup_Singularity"/>
    </widget>
    <addaction name="menuBackup"/>
    <addaction name="separator"/>
    <addaction name="menuRestore"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <addaction name="actionOpen_EVE_data_location"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuUtilities"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBackup_Tranquility">
   <property name="text">
    <string>Tranquility</string>
   </property>
  </action>
  <action name="actionBackup_Singularity">
   <property name="text">
    <string>Singularity (Test Server)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh UI</string>
   </property>
  </action>
  <action name="actionRestore_Tranquility">
   <property name="text">
    <string>Tranquility</string>
   </property>
  </action>
  <action name="actionRestore_Singularity">
   <property name="text">
    <string>Singularity (Test Server)</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionOpen_EVE_data_location">
   <property name="text">
    <string>Open EVE data location</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
